components/brave_perf_predictor/browser/bandwidth_savings_predictor.cc

```diff
@@ -43,11 +43,6 @@ void BandwidthSavingsPredictor::OnPageLoadTimingUpdated(
   if (timing.document_timing->load_event_start.has_value())
     feature_map_["metrics.observedLoad"] =
         timing.document_timing->load_event_start.value().InMillisecondsF();
-
-  // Interactive
```

Comment:
cc @AndriusA

Reply:
all good here, removed the metric from the model, now in master